.buildkite/generate_steps.rb

```diff
@@ -16,6 +16,9 @@ def get_ruby_minor_version(filename, major_version)
   "2.7": {
     "ubuntu20.04": ["amd64", "arm64"],
     "ubuntu18.04": ["amd64"],
+  },
+  "3.0": {
+    "ubuntu20.04": ["amd64", "arm64"]
```

Comment:
arent' there other ruby ubuntu builds? Do we want to add those too? 

which also makes me wonder.....how are the people using those other ruby3.0 ubuntu builds using those builds?

Reply:
I think some people use the internal `fullstaq-ruby`. Which has some more ruby versions associated with it.